pdc_client/__init__.py

```diff
@@ -185,12 +186,45 @@ def obtain_token(self):
                     raise
         raise Exception('Could not obtain token from any known URL.')
 
+    def __call__(self, *args, **kwargs):
+        return self.client(*args, **kwargs)
+
+    def __getattr__(self, name):
+        return self.client.__getattr__(name)
+
+    def __getitem__(self, *args, **kwargs):
+        return self.client.__getitem__(*args, **kwargs)
+
+    def set_comment(self, comment):
+        """Set PDC Change comment to be stored on the server.
+
+        Once you set the comment, it will be sent in all subsequent requests.
+
+        :param comment:     what comment to send to the server
+        :paramtype comment: string
+        """
+        self.session.headers["PDC-Change-Comment"] = comment
+
+
+class PDCClientWithPage(PDCClient):
```

Comment:
Can you hide the new class? E.g. move it to runner.py. I don't think it's useful to anyone but the `pdc` command.